reference/features/configuration.md

```diff
@@ -230,7 +230,7 @@ The `update` database handles [pending updates](/learn/advanced/asynchronous_upd
 **CLI option**: `--http-payload-size-limit`
 **Default value**: `104857600` (~100MB)
 
-Sets the maximum size of accepted JSON payloads. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
+Sets the maximum size of accepted JSON, CSV, or NDJSON payloads. Value must be given in bytes or explicitly stating a base unit. For example, the default value can be written as `107374182400`, `'107.7Gb'`, or `'107374 Mb'`.
```

Comment:
What do you think we find a different way of saying this that doesn't specify accepted file formats? Trying to think ahead and avoiding having to update this when/if we start supporting other formats.

Reply:
Sets the maximum size of accepted payloads. Link this to the Documents page where we mention the acceptable formats?